### PR TITLE
pretext-suite 0.0.1

### PR DESCRIPTION
**Pull request by EdHarry**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

This is a meta-package for all the Pretext Hi-C tools. More tools will be added soon.

### CODE REVIEW

File: recipes/pretext-suite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  version: "0.0.1"
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - pretextmap
+    - pretextsnapshot
```
Comment (mbargull) [inline recipes/pretext-suite/meta.yaml]: Would it makes sense to add version constrains to these packages? Just trying to think ahead here, such that, e.g.,
`pretext-suite 0.0.1` would require `pretextsnapshot <0.1.0a0` and a future `pretext-suite 0.1.0` would require `pretextsnapshot >=0.1,<0.2.0a0`.

**Comment by mbargull** [comment]
> changed my mind...

Totally fine ;).
Just let me know when you're satisfied and I'll merge.

File: recipes/pretext-suite/meta.yaml (hunk 1 above)
Comment (EdHarry) [reply recipes/pretext-suite/meta.yaml]: not really, not right now, the two requirments are a version number appart already,

I can add contraints in the future if i ever make a major change to one of the packages

**Comment by mbargull** [comment]
@bioconda-bot please merge

**Pull request merged**